server.go

```diff
@@ -422,7 +425,9 @@ func (s *Server) Gen(def *Def, r io.Reader, w io.Writer) error {
 
 		// Exists. Write it out and go to the next one.
 		if err == nil {
-			fmt.Fprintln(w, alias)
+			// Flag to indicate that this alias was already set
+			flag := "0"
+			fmt.Fprintln(w, (flag + delimiter + alias))
```

Comment:
Since the delimiter is a space, this can just be `fmt.Fprintln(w, "0", alias)`. See [`fmt.Fprintln` docs](https://golang.org/pkg/fmt/#Fprintln). Concatenation results in a new string and thus memory allocation. We just need to write the string we already have to the `io.Writer`.